src/main/java/com/mcmoddev/lib/container/gui/GuiTextures.java

```diff
@@ -0,0 +1,24 @@
+package com.mcmoddev.lib.container.gui;
+
+import com.mcmoddev.lib.MMDLib;
+import net.minecraft.util.ResourceLocation;
+
+public enum GuiTextures implements IGuiTexture {
```

Comment:
see my reply about gui sprites :)

Reply:
loading them from a JSON and using the enums as 'known keys' would be cool.

Reply:
As discussed on Discord GuiSprite and GuiTextures as enums are fine - your explanation of them being sane defaults is a good one.

Still... a registry sitting on top of these - or replacing them with the defaults they currently provided added automatically to the registry is not a bad idea. It would allow for more configurability - there could be a JSON containing the data that a resource or data pack could provide and it might even allow for less code in MMDLib based mods...